go/conversation/forms.py

```diff
@@ -10,8 +10,9 @@ class ConversationDetailForm(forms.Form):
 class NewConversationForm(ConversationDetailForm):
     def __init__(self, user_api, *args, **kwargs):
         super(NewConversationForm, self).__init__(*args, **kwargs)
+        apps = [v for k, v in sorted(user_api.applications().iteritems())]
```

Comment:
We need the whole source list in-memory for sorting anyway, but making it a generator won't hurt.